tests/framework/web/XmlResponseFormatterTest.php

```diff
@@ -69,6 +69,13 @@ public function formatArrayDataProvider()
                 'c' => [2, '<>'],
                 false,
             ], "<response><a>1</a><b>abc</b><c><item>2</item><item>&lt;&gt;</item></c><item>false</item></response>\n"],
+            [[
```

Comment:
thanks for adding the test. Would be nice to add a comment for understanding that this test is about your usecase